packages/react-components/src/components/VideoGallery/styles/VideoGalleryResponsiveVerticalGallery.styles.ts

```diff
@@ -35,6 +35,8 @@ export const VERTICAL_GALLERY_TILE_SIZE_REM = { minHeight: 6.75, maxHeight: 11,
  *
  * width is being increased by 1rem to account for the gap width desired for the VerticalGallery.
  *
+ * Add 1.5 rem on width to account for horizontal padding
```

Comment:
nit: maybe update comment to: 'Add 1.5 rem on width to account for the horizontal padding on either side of remote video tile.'